klog.go

```diff
@@ -1603,3 +1603,21 @@ func KRef(namespace, name string) ObjectRef {
 		Namespace: namespace,
 	}
 }
+
+// KObjs returns slice of ObjectRef from an slice of ObjectMeta
+func KObjs(arg interface{}) []ObjectRef {
+	s := reflect.ValueOf(arg)
+	if s.Kind() != reflect.Slice {
+		return nil
+	}
+	l := s.Len()
```

Comment:
Nit: there is no benefit to storing s.Len() in variable. It doesn't give performance benefits as I expect that function will be inlined by compiler, and `l` is not great name of variable as it's hard to distinguish from `1` (depends on font you use)

See
```
objectRefs := make([]ObjectRef, 0, l)
objectRefs := make([]ObjectRef, 0, 1)
```